simulator/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM rust:1.70.0-slim-bullseye as builder
+
+RUN apt update && apt install -y \
+    cmake=3.18.4-2+deb11u1 \
+    clang=1:11.0-51+nmu5 \
+    build-essential=12.9 \
+    llvm=1:11.0-51+nmu5
+#RUN rustup target add wasm32-unknown-unknown
```

Comment:
Does the `apt update` affect the version of the dependencies of the software you're installing? Is update really necessary?

Reply:
yeah I'm afraid it is, otherwise apt won't find the dependencies. that first apt update is how apt updates the local database  with the packages available and versions